components/pages/people/case-status/view/case-status.tsx

```diff
@@ -0,0 +1,38 @@
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import Spinner from 'components/Spinner/Spinner';
+import { GetCaseStatus } from 'utils/api/caseStatus';
+import { Resident } from 'types';
+
+interface Props {
+  person: Resident;
+}
+
+const CaseStatusView = ({ person }: Props): React.ReactElement => {
+  const { data: caseStatusData, error } = GetCaseStatus(person.id);
+
+  if (error) {
+    return (
+      <ErrorMessage label="There was a problem with getting case status." />
+    );
+  }
+
+  if (!caseStatusData) {
```

Comment:
this probably isn't an appropriate use of that component, since it's visually much larger than the data it placeholds. either return nothing, or a much smaller spinner/skeleton ui

Reply:
removed spinner